db/db_test.cc

```diff
@@ -4099,6 +4099,42 @@ TEST_F(DBTest, ConcurrentFlushWAL) {
   }
 }
 
+TEST_F(DBTest, ManualFlushWalAndWriteRace) {
+  Options options;
+  options.env = env_;
+  options.manual_wal_flush = true;
+  options.create_if_missing = true;
+  ReadOptions ropt;
+
+  DestroyAndReopen(options);
+
+  WriteOptions wopts;
+  wopts.sync = true;
+
+  port::Thread writeThread([&]() {
+    for (int i = 0; i < 100; i++) {
+      auto istr = ToString(i);
+      dbfull()->Put(wopts, "key_" + istr, "value_" + istr);
+    }
+  });
+  port::Thread flushThread([&]() {
+    for (int i = 0; i < 100; i++) {
+      ASSERT_OK(dbfull()->FlushWAL(false));
+    }
+  });
+
+  writeThread.join();
+  flushThread.join();
+  ASSERT_OK(dbfull()->Put(wopts, "foo1", "value1"));
+  ASSERT_OK(dbfull()->Put(wopts, "foo2", "value2"));
+  Reopen(options);
+  std::string pval;
+  ASSERT_OK(dbfull()->Get(ropt, "foo1", &pval));
+  ASSERT_TRUE(pval == "value1");
+  ASSERT_OK(dbfull()->Get(ropt, "foo2", &pval));
+  ASSERT_TRUE(pval == "value2");
```

Comment:
`ropt` is default-constructed. Here we can simply do `ASSERT_EQ("value1", Get("foo1"))` in which `Get()` is a wrapper of `DB::Get`.

Reply:
Thanks for review. Already use the Get().